var/spack/repos/builtin/packages/font-util/package.py

```diff
@@ -6,11 +6,12 @@
 from spack import *
 
 
-class FontUtil(AutotoolsPackage):
+class FontUtil(AutotoolsPackage, XorgPackage):
     """X.Org font package creation/installation utilities and fonts."""
 
     homepage = "http://cgit.freedesktop.org/xorg/font/util"
-    url      = "https://www.x.org/archive/individual/font/font-util-1.3.1.tar.gz"
+    xorg_mirror_path = "font/font-util-1.3.1.tar.gz"
```

Comment:
@RemiLacroix-IDRIS @tkameyama just in case you added the `font_baseurl`: we could somehow update this to use the selected mirror... (don't know how, cc @adamjstewart)

Reply:
@ax3l: I didn't, @takahiroogura did in https://github.com/spack/spack/commit/e49c101f05a1e784fa906bee2afdc3bd6701c89f.